src/aiida/transports/plugins/ssh_async.py

```
@@ -168,7 +168,7 @@ async def open_async(self):
             raise InvalidOperation('Cannot open the transport twice')
 
         if self.auth_script != 'None':
-            result = subprocess.run(self.auth_script, shell=True, capture_output=True, text=True, check=False)
+            result = subprocess.run(self.auth_script, shell=True, capture_output=True, text=True, check=False)  # noqa: ASYNC221
```

Ruff (0.15.21)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)